### DISCUSSION TITLE
What about adding OpenLLaMA support?

### DISCUSSION THREAD

**Discussion by Sovenok-Hacker** [ggml-org/llama.cpp#1406] [Ideas]
I think this is a very interesting model. The quality is a bit worse than the original LLaMA, but it can be used for commercial purposes. It took me about 2 hours to get it working with LLaMA.CPP. I used code from [this gist](https://gist.github.com/MillionthOdin16/022f47465c88cbf2db562d88d2baa884) to convert [official files](https://huggingface.co/openlm-research/open_llama_7b_preview_200bt) to GGML format and quantized it. It is a good and growing model. I think it would be good to add a guide on how to run it.

**Comment by Green-Sky** [comment]
You must have done something wrong, since it should just have worked.
see https://github.com/ggerganov/llama.cpp/issues/1291

**Reply by Sovenok-Hacker** [reply]
Ok
